python/src/py/obj.h

```diff
@@ -90,6 +90,7 @@ static inline bool mp_obj_is_qstr(mp_const_obj_t o)
     { return ((((mp_int_t)(o)) & 3) == 2); }
 #define MP_OBJ_QSTR_VALUE(o) (((mp_uint_t)(o)) >> 2)
 #define MP_OBJ_NEW_QSTR(qst) ((mp_obj_t)((((mp_uint_t)(qst)) << 2) | 2))
+mp_obj_t mp_obj_new_int_via_str(const char* value, int base);
```

Comment:
We try to avoid changing the external code. The content of python/src/py is identical to the MicroPython repository. This code should go somewhere in python/src/port/.

Reply:
I thought this was the better place :/ I will try to move it in port.cpp